packages/MonorepoBuilder/packages/Split/src/PackageToRepositorySplitter.php

```diff
@@ -59,8 +59,11 @@ public function __construct(
     /**
      * @param mixed[] $splitConfig
      */
-    public function splitDirectoriesToRepositories(array $splitConfig, string $rootDirectory): void
-    {
+    public function splitDirectoriesToRepositories(
+        array $splitConfig,
+        string $rootDirectory,
+        int $maxProcesses = 0
```

Comment:
This should be required value as well, to keep API simple and un-ambiguous

Reply:
Since the cli option no longer comes with a default value, I changed this to an optional parameter defaulting to `null`: 266f99baf